submissions/xT0PERx/js-dom/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="visuality.css" />
+    <title>My interactive side-menu</title>
+  </head>
+  <body>
+    <div class="container clearfix">
+      <div class="menu">
```

Comment:
Try to use more semantic tags, it helps search bots to figure out what each block is responsible for.
For navigation use `nav tag`.

Reply:
Fixed